src/Runtime.php

```diff
@@ -4,20 +4,30 @@
 
 abstract class Runtime
 {
+    public const NODEJS_20 = 'nodejs20.x';
+
+    public const NODEJS_18 = 'nodejs18.x';
+
     public const NODEJS_16 = 'nodejs16.x';
 
     public const NODEJS_14 = 'nodejs14.x';
 
-    public const NODEJS_12 = 'nodejs12.x';
+    public const PYTHON_311 = 'python3.11';
```

Comment:
Can add Python 3.12 as well. 👍🏻
```suggestion
    public const PYTHON_312 = 'python3.12';

    public const PYTHON_311 = 'python3.11';
```

Reply:
Can you point me to when was Python 3.12 added? So I can update this PR with this

Reply:
https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#214960